src/components/App/Dashboard/Balances/BalanceItem/BalanceItem.js

```diff
@@ -17,31 +18,56 @@ class BalanceItem extends Component {
   static propTypes = {
     amount: PropTypes.number,
     asset: PropTypes.string,
+    history: PropTypes.arrayOf(
+      PropTypes.shape({
+        close: PropTypes.number.isRequired,
+        high: PropTypes.number.isRequired,
+        low: PropTypes.number.isRequired,
+        open: PropTypes.number.isRequired,
+        time: PropTypes.number.isRequired,
+        volumefrom: PropTypes.number.isRequired,
+        volumeto: PropTypes.number.isRequired,
+      })
+    ).isRequired,
     usdAmount: PropTypes.number,
     usdPrice: PropTypes.number,
   };
 
+  state = {
+    isOpened: false,
+  };
+
+  clickHandler = () => {
+    this.setState({ isOpened: !this.state.isOpened });
+  };
+
   render() {
     const {
-      asset, amount, usdAmount, usdPrice,
+      asset, amount, usdAmount, usdPrice, history,
     } = this.props;
 
     return (
-      <Container>
-        <CurrencyIcon name={`currency${capitalize(asset)}`} size={40} />
-
-        <AssetInfo>
-          <Asset>{asset.toUpperCase()}</Asset>
-          <Price>${usdPrice.toFixed(6)}</Price>
-        </AssetInfo>
-
-        <AmountInfo>
-          <UsdAmount>${usdAmount.toFixed(6)}</UsdAmount>
-          <Amount>
-            {amount} {asset.toUpperCase()}
-          </Amount>
-        </AmountInfo>
-      </Container>
+      <TouchableOpacity onPress={this.clickHandler}>
+        <Container margin={this.isOpened && history !== null}>
+          <CurrencyIcon name={`currency${capitalize(asset)}`} size={40} />
+
+          <AssetInfo>
+            <Asset>{asset.toUpperCase()}</Asset>
+            <Price>${usdPrice.toFixed(6)}</Price>
+          </AssetInfo>
+
+          <AmountInfo>
+            <UsdAmount>${usdAmount.toFixed(6)}</UsdAmount>
+            <Amount>
+              {amount} {asset.toUpperCase()}
+            </Amount>
+          </AmountInfo>
+        </Container>
+        {this.state.isOpened
```

Comment:
I think it better to destruct state like props in the first lines of this function if we use `this.state.isOpened` several times.